build.gradle

```diff
@@ -75,11 +75,6 @@ test {
         systemProperty "pipeline.stack.write", project.getProperty("pipeline.stack.write")
     }
 
-    jacoco {
-        classDumpDir = file("$buildDir/jacoco/classpathdumps")
-    }
-
-    finalizedBy jacocoTestReport
```

Comment:
Why you no like Jacoco?

Reply:
Doh, debugging at its finest bleeding over.